osu.Game/Screens/Edit/Timing/DifficultySection.cs

```diff
@@ -18,7 +19,7 @@ private void load()
             {
                 multiplierSlider = new SliderWithTextBoxInput<double>("Speed Multiplier")
                 {
-                    Current = new DifficultyControlPoint().SpeedMultiplierBindable,
+                    Current = new DifficultyControlPoint().SliderVelocityBindable,
```

Comment:
Kind of a drive-by place to put it but effect points probably need UI for specifying approach rate, too, at some point. Although I'm not sure how to best explain what the difference is (I'm not entirely sure I have a good grasp on it myself yet).

Reply:
Has been added.